codice/ddf
# DDF-1955: Refactors AdminSourcePollerBean into its own module

#### What does this PR do?
AdminSourcePoller has now been moved into its own module and has also been renamed to reflect that it is becoming more general. 
A publish method was added for registry capability. 
#### Who is reviewing it?
@clockard @gordocanchola @mcalcote @brianfelix 
#### How should this be tested?
Verify build with i-tests. 
Once, https://github.com/codice/ddf/pull/662 is merged, some more verification will be required. 
#### Any background context you want to provide?
This refactor will be important when adding more registry things to this poller. 
#### What are the relevant tickets?
DDF-1955
Depends on DDF-1831 https://github.com/codice/ddf/pull/662
#### Checklist:
